central/globaldb/postgres.go

```diff
@@ -187,6 +187,10 @@ func CollectPostgresStats(ctx context.Context, db *postgres.DB) *stats.DatabaseS
 		statsSlice = append(statsSlice, tableStat)
 	}
 
+	if rows.Err() != nil {
```

Comment:
if err := rows.Err(); err != nil

Then can use this instead of rows.Err() below